### PR TITLE
Interview assignment edits

### PR DESCRIPTION
**Pull request by ritsok**

### CODE REVIEW

File: education-engineer/assignment.md [hunk 1]
```diff
@@ -1,44 +1,33 @@
...
-
-You may link to existing resources created by HashiCorp or the community, but do not plagiarize content from HashiCorp tutorials or documentation. 
+Please take the time to carefully review the [writing style guide](../styling-guide-snippet.md) and the [guide template](../guide-template.md) before getting started.
 
-### Get started
-
-We recommend you clone the project to your local workstation. Use the Code button on the main project page to retrieve the URL and issue the `git clone` command on your workstation.
-
-Alternatively, you may download the [terraform-getting-started.md](terraform-getting-started.md) file. This file is located in the `education-engineer` folder. You can also download the entire project as a zip.
+## Instructions
```
Comment (judithpatudith) [inline education-engineer/assignment.md:7]:
```suggestion
## Instructions

```

File: education-engineer/assignment.md [hunk 2]
```diff
@@ -1,44 +1,33 @@
...
+Update the [sample text](terraform-getting-started.md) to follow the provided [writing style guide](../styling-guide-snippet.md) and
+[guide template](../guide-template.md).
 
-**NOTE**: DO NOT FORK THE PROJECT. MAKE A COPY OF ALL FILES, CREATE YOUR OWN FRESH REPOSITORY AND SUBMIT A PULL REQUEST TOWARDS YOUR OWN PROJECT. DO NOT MERGE THE PULL REQUEST.
+**Any changes that you think will improve the text and explain the concepts better are welcome**. If anything in the text does not match your opinion on a best practice, feel free to correct the meaning of the text. You may link to existing resources created by HashiCorp or the community, but do not plagiarize content from HashiCorp tutorials or documentation. 
 
-### Edit the sample
+To complete the assignment:
+- Create a new GitHub repository for your submission. Make sure the repository is publicly accessible for our team to review your submission.
```
Comment (judithpatudith) [inline education-engineer/assignment.md]:
```suggestion
- Create a new, public GitHub repository for your submission (do not fork this repository). Make sure the repository is publicly accessible so we can review your submission.
```

File: education-engineer/assignment.md [hunk 3]
```diff
@@ -1,44 +1,33 @@
...
+- Before editing, commit [the initial version of the sample text](terraform-getting-started.md) to the main branch of the repository. 
+- Create a new branch and update the sample text to satisfy the style guide.
+- Create a pull request for your proposed changes to the text.
```
Comment (judithpatudith) [inline education-engineer/assignment.md]:
```suggestion
- Create a pull request for your proposed changes to the text. Do not merge the pull request.
```

File: education-engineer/assignment.md [hunk 4]
```diff
@@ -1,44 +1,33 @@
...
 
-Now that your environment is set up, you can start to edit the file [terraform-getting-started.md](terraform-getting-started.md) on your local machine. Complete the following tasks, make all changes in Markdown.
+**NOTE: DO NOT FORK THE PROJECT.** Make a copy of all files, create a fresh repository, and open a pull request against your own project. Do not merge the pull request.
```
Comment (judithpatudith) [inline education-engineer/assignment.md]:
```suggestion
**NOTE: DO NOT FORK THE PROJECT.** Make a copy of all files, create a fresh repository, and open a pull request against your own project.
```

File: styling-guide-snippet.md [hunk 5]
```diff
@@ -1,30 +1,23 @@
...
 ## Why?
 
-A consistent writing style and voice makes the HashiCorp brand
-strong and vibrant. There are a few reasons for our writing
-style choices. In particular, we want to optimize for:
+A consistent writing style and voice strengthens the HashiCorp brand and 
+produces clearer educational materials. Our style guide optimizes for:
 
-- **Non-native speakers** - Literal translations do not mean the same thing in different countries or regions.
+- **Non-native speakers** - Idiomatic language may not translate easily in different countries or regions.
```
Comment (judithpatudith) [inline styling-guide-snippet.md]:
```suggestion
- **Non-native speakers** - Common figures of speech may not translate easily in different countries or regions.
```

File: styling-guide-snippet.md [hunk 6]
```diff
@@ -142,9 +135,9 @@ make sure they are necessary, and if so, use the gender neutral "they/them".
 -|-
 |Once you give the developer the token they can...|Once you give the developer the token she can...|
 
-### Over Simplification
+### Oversimplification
 
-Our products can be complicated. Words that indicate that processes are easy
+Our products can be complex. Words that indicate that processes are easy
 don't can alienate users who are having a hard time or are encountering bugs.
```
Comment (judithpatudith) [inline styling-guide-snippet.md]:
```suggestion
can alienate users who are having a hard time or are encountering bugs.
```

**Review by judithpatudith — APPROVED** [review]
I think this is much nicer, thank you! 

@im2nguyen @yhyakuna and @topfunky is it possible for you to review these changes tomorrow so we can get this out to some candidates I have in the pipeline? If you have other improvements you see while reviewing I think we can probably take follow up PRs. This is just such an improvement that I'd like to avoid sending out the old version if we can. Thanks!

File: README.md [hunk 7]
```diff
@@ -1,11 +1,12 @@
...
-For questions or concerns please reach out to the HashiCorp Education team at `#team-education` or through team-education@hashicorp.com 
+
+This project contains interview assignments for roles on the HashiCorp Education team.
+Contact the team at `#team-education` or team-education@hashicorp.com with any questions or concerns.
 
 
 # Assignments
 
-Click on the link for the respective role you are applying for below to view the writing assignment. Make sure to review the writing style guide and the layout guide prior to starting the assignment.
+Select the role you are applying for from the list below to access the corressponding writing assignment. Make sure to review the writing style and the layout guides prior to starting the assignment.
```
Comment (im2nguyen) [inline README.md]:
```suggestion
Select the role you are applying for from the list below to access the corresponding writing assignment. Make sure to review the writing style and the layout guides prior to starting the assignment.
```

**Review by yhyakuna — APPROVED** [review]
Thanks for making those changes.

**Review by im2nguyen — APPROVED** [review]
thank you!

**Pull request merged**